### PR TITLE
cleanup a few warnings, add first draft info page issue #20

### PR DESCRIPTION
**Pull request by lmccart**
Here's a first draft of the info page. Todo:
* Work on the page height / scrolling
* We need the final curatorial statement
* We need some design thought about how to fit this much text (how should scrolling work with this layout?) and where does the cursor go in all this?

https://github.com/blerchin/dma-20-mfa-show/issues/20

### CODE REVIEW

File: src/Containers/App/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import React, { Suspense, lazy } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import './style.css';
 import BlobField from 'src/BlobField';
+import Info from 'src/Containers/Info';
```
Comment (lmccart) [inline src/Containers/App/index.js:6]: not sure if this should go in `src/Containers`, `src/Components`, or above that on the same level as `BlobField/`

Comment (brysonian) [reply src/Containers/App/index.js:6]: I think containers. Generally those are pretty much equivalent to "pages" BlobField should probably go in components if we wanted to be very buttoned up.

**Comment by blerchin** [comment]
btw there is some initial work on this on the `info-page` branch. Might not be any further along than this one though :-)

**Comment by lmccart** [comment]
ah ok, feel free to close this or pull code from it, or whatever you'd like. it looks like there are two different implementations of the clipping, not sure which one has better result. @cleivas I'll follow your lead..

**Comment by blerchin** [comment]
I think Clara was actually hoping other people would pick up on the work and finish it. Didn't mean to discourage more work on this PR! It looks like the way you are updating the circle shape is more performant than using react state like we were doing on that other branch. I would suggest using `refs` to get a handle to the element rather than hardcoding the ID, though

**Comment by dalena** [comment]
Wow I didn't even realize this info page was made. It looks really good. I'm wondering why there are two versions of the info page and which one is current? There's a pull request for a different version of the info page btw. The deploy preview for this one is the closest to the mockup.

There are some overflow-y issues when the circle goes beyond the frame at the bottom. Also think we should probably just remove the title from the back, increase circle size, and narrow the container width for better legibility.

One general concern I have is that conceptually it distracts from our main blob feature. This circle clipping is a novel interaction idea but may not be great for reading longer form text. Can some more people weigh in on this with me? @lmccart  @blerchin @hsab @brysonian 

I do think it is still a great feature and don't want to dispose of the idea completely. Maybe we can have the info page be a very vanilla "text on a page" and use this clipping feature for something else after launch? Maybe paragraphs from our theses or little messages about 2020 that people can "discover" or something we can look back at a few years for a little pang in our hearts :~)

**Comment by lmccart** [comment]
@dalena I think @cleivas and I both started versions of this without realizing. @hsab created a PR for the other branch so we could easily preview both. I think this one is a little closer and more performant, but please use whichever is useful for you.

I think I agree about the circle competing visually with the blobs. But I leave that to you all.

One other question, there is currently no link from the homepage to the info page. Do we want one? It seems like the curatorial statement as framing device is lost a bit if everyone goes straight into artist pages.

**Comment by dalena** [comment]
@lmccart I would say yes to the nav on homepage. I think the nav should be on every page! I'll create an issue for that

**Comment by hsab** [comment]
I wanted to refrain from commenting on this given the time and effort put into it, but since Dalena brought it up, I thought I'll chime in as well.

I like the novelty of the circle/text interaction, but I find it a bit too much although it was proposed in our mockups. I'm very much aligned with a vanilla text on page. Advantage is that we can reuse all of our components and don't have to worry about accessibility, interaction, styling, debugging etc.

My other concern was the visual difference between the circle and the blobs. I tried looking at options for having the path animated. Similar to how we have the blobs, but most of the solutions I found required either animating the svg path which didn't seem ideal, or embedding the text in a canvas element which is not really good for seo, styling, and accessibility. 

I really appreciate the time and effort :pray:  but I'm voting for a single column "vanilla" text on page.

**Comment by lmccart** [comment]
yeah all good with me, wasn't too much time!

**Comment by blerchin** [comment]
Yeah, I agree the circle isn't necessary and creates a lot of problems. I don't think we need to distract from the writing.
Also agreed about the Nav on the front page, thanks for bringing that up Lauren!

**Comment by hsab** [comment]
yes, info def needs to be on the homepage. I was thinking of having it on the top left, so it never interferes with the blobs regardless of the device orientation. 

These are bad mockups, but the idea is same font/size/weight/capitalization as nav menu, but positioned as the project header's artist line. There's not point in having `HOME` one homepage right? 

Screenshot_20200525_152910
Screenshot_20200525_152919

**Comment by lmccart** [comment]
cool, I think we can close this and #50 then.
@dalena are you planning to work on the info page, or would you like me to?
is there a final curatorial text ETA?

**Comment by dalena** [comment]
@hsab it's not good design practice to have navs in different places. We should probably keep to the left bottom corner and constrain the divs as much as possible. Considering the surface area of the blobs in relation to the nav text, I don't think that's going to be a problem. 

@lmccart if you have spare time before thursday morning to work on it, that would be helpful. Otherwise, I'll be free on thursday to work on web stuff again. As for the final curatorial text, the ETA is actually the 29th, which is unfortunate. But I just asked Ana if there's any possibility it can be sooner or at least give us a draft that has a similar length to the expected final. I'll get back with details.

**Pull request closed without merging**